docker-compose/.env

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Should we push a .env.test file instead?

Reply:
do you see some other way of using it? For now, we don't have to provide `--env-file` arg for docker-compose command, as that's a default name